source/fundamentals/typescript.txt

```diff
@@ -128,7 +128,7 @@ must manually check that your nested field values have your intended type.
    when specifying subdocuments in a query. Dot notation allows you to query
    nested fields without matching subdocuments exactly.
 
-   To show this behavior, lets say you have a collection containing
+   To show this behavior, let's say you have a collection containing
```

Comment:
Suggestion: maintain formality in voice by avoiding "let's say"

"To show this behavior, consider a collection composed of the following document:"